src/context/internal/uicontextresolver.cpp

```diff
@@ -161,6 +161,10 @@ bool UiContextResolver::isShortcutContextAllowed(const std::string& scContext) c
     //! NotationOpened
     //!     NotationFocused
     //!         NotationStaffTab
+    //!
+    //! UPDATE I'm now adding one more context for list VS range selection, but this is
+    //! quite clearly not an optimal solution. In future, we need a general system to
+    //! allow/disallow shortcuts based on any property of the currentNotation. [M.S.]
```

Comment:
Perhaps we should bind it to enabledness of actions, because it looks like that's currently not the case. However, that's not the whole story though, because sometimes both actions are enabled and then we still need to figure out which shortcut has precedence.

Reply:
Yeah, this is a place that needs some rethinking imo